hack/release.go

```diff
@@ -133,7 +135,7 @@ func main() {
 		}
 		// Generate new file full path
 		outputFile := filepath.Join(*outputDir, strings.Replace(filepath.Base(file), ".template", ".yaml", 1))
-		f, err := os.Create(outputFile)
+		f, err := os.Create(filepath.Clean(outputFile))
```

Comment:
There's three instances of adding filepath.Clean()  to file operations which was caught by the `gosec` linter.